### DISCUSSION TITLE
Add some documents when a recipient fill the PDF

### DISCUSSION THREAD

**Discussion by Sorcier** [docusealco/docuseal#92] [Ideas]
It could be practical when a recipient fills in the PDF that he can add attachments as a page (and not a link to the documents) for example an expense report that we would fill in and to which we would add the invoice to form a single multi-page PDF.

**Comment by AlexBTurchyn** [comment]
Hi @Sorcier that's for the suggestion - it totally makes sense but that's something that differs from basign document signing and unfortunetely we can't focus on developing it right now - but hopefully something like that will be available later.

But I think it's possible to achieve the same results currently with just a single manual step for you.

So you need to collect their documents via the 'file' field.
Then you need to download this file from the user submission results and upload it into the template as a new document.
And then you just need tp draw your specific fields and submit the template form with this new document and fields.

Hope this helps - let me know the the solution described above works for you.
